### DISCUSSION TITLE
Problem Viewing Published App on Macbook / Safari

### DISCUSSION THREAD

**Discussion by chadgriff** [Budibase/budibase#4489] [Help] [answered]
I have been gradually deploying apps within my organization and experienced an issue today for the first time. One of my users was getting the attached error message when attempting to view a published app on his Mac Book using the Safari browser.  There are many other users who can access the app (either via a PC or a Mac) just fine.  This one particular Mac Book was running High Sierra 10.13.6 OS with Safari browser version 13.1.2.  I republished the app to his personal user account  and also republished the app itself out to all users. No change in behavior...he still gets the same error message. He can successfully get to the app via his iPhone so that tells me his user account within Budibase is in good shape.  Any ideas or guidance for how to interpret this error message would be appreciated. 
![safari_error_message](https://user-images.githubusercontent.com/15015476/153922841-b5d18e9b-c13e-487a-8d25-d706efd06a46.png)

**Comment by mjashanks** [comment]
Thanks Chad - we are looking into this.

**Comment by micebrain** [comment, marked as answer]
Hello @chadgriff 
I don't know which version is your app but I believe this might be the same issue as I reported earlier - https://github.com/Budibase/budibase/issues/4452
The latest release fixed this. You might want to try it.
